src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -664,6 +690,10 @@ SubCmdProgram::execute(const SubCmdOptions& _options) const
                                                                          "  SC    - Satellite controller"*/)
     ("user,u", boost::program_options::value<decltype(xclbin)>(&xclbin), "The xclbin to be loaded.  Valid values:\n"
                                                                       "  Name (and path) of the xclbin.")
+    ("image", boost::program_options::value<decltype(image)>(&image)->multitoken(), "Specifies an image to use used to update the persistent device.  Value values:\n"
+                                                                    "  Name (and path) to the mcs image on disk\n"
+                                                                    "  Name (and path) to the xsabin image on disk\n"
+                                                                    "Note: Multiple images can be specified separated by a space")
```

Comment:
Lets leave this description out.
It might be best if they simply use --image twice.  For example:
`--image primary.msc --image secondary.mcs.`